operator/config/samples/component-integration-installed/secret-gardener-k8s.yaml

```diff
@@ -0,0 +1,10 @@
+apiVersion: v1
+kind: Secret
+metadata:
+  name: kyma-sample
+  namespace: default
+  labels:
+    "operator.kyma-project.io/kyma-name": "kyma-sample"
+    "operator.kyma-project.io/managed-by": "kyma-operator"
+data:
+  config: YXBpVmVyc2lvbjogdjEKa2luZDogQ29uZmlnCmN1cnJlbnQtY29udGV4dDogc2hvb3QtLWplbGx5ZmlzaC0tam0tdGVzdC0xCmNvbnRleHRzOgogIC0gbmFtZTogc2hvb3QtLWplbGx5ZmlzaC0tam0tdGVzdC0xCiAgICBjb250ZXh0OgogICAgICBjbHVzdGVyOiBzaG9vdC0tamVsbHlmaXNoLS1qbS10ZXN0LTEKICAgICAgdXNlcjogc2hvb3QtLWplbGx5ZmlzaC0tam0tdGVzdC0xLXRva2VuCmNsdXN0ZXJzOgogIC0gbmFtZTogc2hvb3QtLWplbGx5ZmlzaC0tam0tdGVzdC0xCiAgICBjbHVzdGVyOgogICAgICBzZXJ2ZXI6ICdodHRwczovL2FwaS5qbS10ZXN0LTEuamVsbHlmaXNoLnNob290LmNhbmFyeS5rOHMtaGFuYS5vbmRlbWFuZC5jb20nCiAgICAgIGNlcnRpZmljYXRlLWF1dGhvcml0eS1kYXRhOiA+LQogICAgICAgIExTMHRMUzFDUlVkSlRpQkRSVkpVU1VaSlEwRlVSUzB0TFMwdENrMUpTVU0xZWtORFFXTXJaMEYzU1VKQlowbFNRVXh2VG5CclQwbGtkSE5pUjJaaWVuSnpUMmRqWWpCM1JGRlpTa3R2V2tsb2RtTk9RVkZGVEVKUlFYY0tSRlJGVEUxQmEwZEJNVlZGUVhoTlExa3lSWGRJYUdOT1RXcEpkMDVxU1hoTlJHTjVUVlJGTUZkb1kwNU5la2wzVG1wSmVFMUVZM2xOVkVVd1YycEJUZ3BOVVhOM1ExRlpSRlpSVVVSRmQwcHFXVlJEUTBGVFNYZEVVVmxLUzI5YVNXaDJZMDVCVVVWQ1FsRkJSR2RuUlZCQlJFTkRRVkZ2UTJkblJVSkJTamRJQ21WblN6VlBXbG8wZDJGcWRVRTNPRUZwVTNZNFdWTjVVblZ0VG5aeFFVNWpVRkYwUkRSdFFsbENOVll2VUZoME5uZHROV05uZGxoeWRXTXlOa3h6ZVRrS2EzbGxWalZxU2trNU1GTjNjR1pxVEdWUVJGbHhaV3RJTm5SUGNpOXFWR3RYTkdSQk5sWkxZVmxsZGtWVWJrSlVVWGxZV0V0VldWWmpXR1Y0VjJ0MFZRcGhURkZvVlhWamNUVkphR05wVm5Gak5HaEJhR1JVY1RCQ05GTTBTbkJoVG1ONE1uUTNUakYyTTJOcFYwcGlVMHd3U25kMU1FVkpha3RvU1dkblNYaEpDbVpPV2tKSVMzUkhjbE5rZW1SdFJGWnpNV0pyTUZaMU4ydEllbkpaVGxsU1MzSmFSamRxVmtzM2VGVXJlRFZCTURVNVRtZENlazExT0ZCWlFVWTNNRElLTDFCa05sQXpUVTQwVERJMlRtOXdjelp2YmpaWE9WUm1la3BCTW10WGVrRm9UV1ZxVVVKb2IzSnFOa3R2V1U0dmRVbzNhRmRrYjFnMWRrSXpWRXc1VkFwSlREVjNhVGN6VUd0V05pdDNNR3RNVjFCalEwRjNSVUZCWVU1RFRVVkJkMFJuV1VSV1VqQlFRVkZJTDBKQlVVUkJaMGR0VFVFNFIwRXhWV1JGZDBWQ0NpOTNVVVpOUVUxQ1FXWTRkMGhSV1VSV1VqQlBRa0paUlVaR1dGWXJkblZUTWxOVlRXWjJZazlxVmpGblJITTJkMWRRZFRSTlFUQkhRMU54UjFOSllqTUtSRkZGUWtOM1ZVRkJORWxDUVZGQ2NYTTJZVzFHYUZaUFpVaFRSRVZNYzFOR1JEa3JkRGs1U1hOa05EbFJhVzlIY2t4aEx6Z3ZlR2wzZDFGcVFpczJjZ3BwT1VSWVVGVktNR2s1UlcxVk5XTkNaa2RUZW1aSFRqRXhhbEZPVVc1VWVrTlJVakV2VUc5bFRqTk5hMHd3YkVkSGRtbFhXV3AzY1ZkWVprMVFhbG9yQ21rMk9YWlJXVlp5UWxGWFRubGlVMjl6VVVoMWRYTnliWEpYV0ZSQ2JFaGtORWhyY0d4M1RUbFhiMHMzZWt0MVZFUm1Sa0pzVDFSRE9XcDVXVE4xTWpFS1pVaEthM2xoV20wNVFYa3JWbXRtVDA5eGNsSkJVelZuYVZkVlNGSmpVMnAwVnpCT2FtbERibkY0ZUU1QlQxZzBWbFZaT0ZkUldtbElZM0ZJVTBsRFdncFdiVzF5WnpSME5sZDFaRWRQWTNWcWRFODRiakptT1RsdkswdzVjVWwxYVZoRFZYTmxiazlYU1RSR2QzUk1Ra05XWmtKaWIycFVkR1ZSS3pZelMxUXZDbUp0ZGxRMVJHOVlLMEowU2twMmRWRlpkRzFHUVhwSGJFWkRUSGQxVEZWUVJGVm9aQW90TFMwdExVVk9SQ0JEUlZKVVNVWkpRMEZVUlMwdExTMHRDZz09CnVzZXJzOgogIC0gbmFtZTogc2hvb3QtLWplbGx5ZmlzaC0tam0tdGVzdC0xLXRva2VuCiAgICB1c2VyOgogICAgICB0b2tlbjogPi0KICAgICAgICBwR1hUVnpFU2MzR1F4SHNaSk1pTEVWT0FOWXYwU1ExdUlnOXZTVkdhVk0zOTBqbm51N1duYmFXcEgwOVZKbU5oM2RqeDZRdzU5cVRWNko0cVhhRXplS0E4OGZmc2hTN2JpeURoNHhCRGZHRnZ4S3NUcWZzWE81NzFwRFo4UTZOdQoK
```

Comment:
this is an already rotated config for sample purposes, its not actually used anywhere and we can keep it.